src/generator/www/js/main.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+  let $email =$("#email");
+  let $name  =$("#name"); 
+  let $url  =$("#url");
+
+  $("#btn").click(function() { 
+
+  var form = document.querySelector("form");
+  var fd = new FormData(form);
+
+  var files ={
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l12)